Mathlib/Algebra/Order/Ring/Unbundled/Rat.lean

```diff
@@ -0,0 +1,224 @@
+/-
+Copyright (c) 2019 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro
+-/
+import Mathlib.Algebra.Order.Group.Unbundled.Abs
+import Mathlib.Algebra.Order.Group.Unbundled.Basic
+import Mathlib.Algebra.Ring.Rat
+import Mathlib.Init.Data.Int.Order
+
+/-!
+# The rational numbers possess a linear order
+
+This file constructs the order on `ℚ` and proves various facts relating the order to
+ring structure on `ℚ`.
+
```

Comment:
Can you allude to the `Unbundled` name in this docstring? Perhaps "This file only provides the unbundled `CovariantClass` instance; the bundled `LInearOrderedField` instance is left to a later file".

Reply:
Done